handbook/people-ops/hiring/evaluating_values.md

```diff
@@ -0,0 +1,29 @@
+# Evaluating values in an interview
+
+We expect every teammate, team, and our company to live up to the [Sourcegraph values](../../../company/values.md). It is therefore important taht we evaluate every candidate on the values to determin how successful they will be within Sourcegraph.
+
+It is the responsibility of every interviewer to think about these values when interviewing a candidate. When submitting feedback for a candidate, copy the list of values into your notes and score the candidate out of 5 based on the rubric.
+
+## Rubric
+
+Scored out of 5:
+
+- 1 - Exhibited behavior that is contrary to this value
+- 2 - I did not have any evidence towards or against this value (neutral)
+- 3 - Gave the impression they have this value, but lacking evidence
+- 4 - Demonstrated this value with evidence
+- 5 - Exceptionally demonstrated this value, with abundent evidence
+
+## Values to copy into feedback
+
+Use this plaintext format for easy addition to the Lever feedback area.
+
+```markdown
+- Be customer driven:
+- Work as a team:
+- High agency:
+- High quality:
+- Open and transparent:
```

Comment:
Does good communication fall under this?

Reply:
Yes, I think so. Where can we clarify this?

Reply:
My instinct is that we should come up with rubric that is informed by our values, but not necessarily exactly the values 1-1, because that will make it easier to evaluate candidates.

For example:

- Customer(/outcome/metrics?) driven
- Collaborative (I think this covers both work as a team and be a good human)
- High agency
- Iterative
- Effective communication (clear, succinct)
- Growth mindset
- Authentic and kind

Reply:
After setting this aside for a bit, I actually think feeling like we need to move away from our values to other rubrics means that our values are not quite right. We should live by and evaluate base on our values and use those as the rubric, otherwise this will lead to confusion and misalignment over time.

Reply:
I am fine with that. I think just having more alignment/training/details on what these value mean will help (in the values handbook page)

Reply:
I completely agree. I don't think it hurts to add this change (or the questions) in the meantime while we work on the RFC proposal to get that in place. Perhaps adding a note here to indicate that the work you mention is in progress?